plasma/client/child_chain_service.py

```diff
@@ -0,0 +1,48 @@
+import requests
+import json
+import rlp
+from plasma.child_chain.child_chain import ChildChain
+from plasma.child_chain.transaction import Transaction
+from plasma.child_chain.block import Block
+
+
+headers = {'content-type': 'application/json'}
+
+
+class ChildChainService(object):
+
+    def __init__(self, url):
+        self.url = url
+        self.methods = [func for func in dir(ChildChain) if callable(getattr(ChildChain, func)) and not func.startswith("__")]
+
+    def send_request(self, method, args):
+        payload = {
+            "method": method,
+            "params": args,
+            "jsonrpc": "2.0",
+            "id": 0,
+        }
+        response = requests.post(self.url, data=json.dumps(payload),
```

Comment:
Easier to use `requests.post(self.url, json=payload).json()`. This also removes the need for the additional `Content-Type` header.